anvil/src/x11.rs

```diff
@@ -300,9 +275,28 @@ pub fn run_x11(log: Logger) {
                 cursor_visible = true;
             }
 
-            pointer_element.set_position((x as i32, y as i32));
+            let scale = Scale::from(output.current_scale().fractional_scale());
+            let cursor_pos = state.pointer_location;
+            let cursor_pos_scaled = state.pointer_location.to_physical(scale).to_i32_round();
+
+            pointer_element.set_position(cursor_pos.to_i32_round());
```

Comment:
Seems like the `PointerElement` position is now unused because it is no longer is a space element.